TESTING.md

```diff
@@ -2,27 +2,29 @@
 
 This document aims to provide some basic information on writing and running the `nvim-treesitter` test suite.
 
-## Running tests
+## Setup
 
-### Setup
+### Install plenary and nvim-treesitter dependencies
 
-The only required dependency to run `nvim-treesitter` tests is `plenary.nvim`: https://github.com/nvim-lua/plenary.nvim
+You'll need to have modules `plenary.nvim` and `nvim-treesitter` available on your runtime path.
 
-#### Installing plenary.nvim using packer
+If you use packer, plenary should be on the runtime path already. Otherwise, the simplest way to get it is to add a copy of the `plenary.vim` file to your local `plugin/` directory.
 
-TODO: I'm not sure how to set this up.
+Here's a quick solution to get latest.
 
-#### Installing plenary.nvim using vim-plug
+```
+wget https://raw.githubusercontent.com/nvim-lua/plenary.nvim/master/plugin/plenary.vim -O plugin/plenary.vim
```

Comment:
does windows have wget? curl should be the preferred one here imo

You need to have the entirety of `plenary.nvim` in `&rtp`, not just plenary.vim